src/resources/app/components/AnnouncementContent.vue

```diff
@@ -101,7 +110,8 @@
 </template>
 
 <script>
-import {EditAnnouncement} from "../../api/announcement/EditAnnouncement";
+import {editAnnouncementAPI} from "../../api/announcement";
+import {deleteAnnouncementAPI} from "../../api/announcement";
```

Comment:
can be written into  one statement,
```suggestion
import { deleteAnnouncementAPI, editAnnouncementAPI } from "../../api/announcement";
```